README.md

```diff
@@ -2,7 +2,7 @@
 <!-- CC-BY-SA-4.0 AND Apache-Patent-License -->
 <!-- See LICENSE.md file for details -->
 <div align="center">
-   <img id="index_logo" src="Arm_logo_blue_RGB.svg" />
+   <img id="index_logo" src="{{site.baseurl}}/assets/images/Arm_logo_blue_RGB.svg" />
```

Comment:
The logo doesn't show up anymore: https://github.com/ValeriaDaneva/acle/tree/github.io-correct-image-file

In fact, I'd be surprised if this would be working because there is no "Liquid" tags here. If I hover on the broken image, it is targeting this link: https://github.com/ValeriaDaneva/acle/blob/github.io-correct-image-file/%7B%7Bsite.baseurl%7D%7D/assets/images/Arm_logo_blue_RGB.svg

This means that `{{site.baseurl}}` is not interpreted by Jekyll.
![Screenshot 2021-12-01 at 14 28 36](https://user-images.githubusercontent.com/25690309/144252741-d81af67a-f50d-48f0-be66-657b54684a6c.png)